setup.sh

```diff
@@ -200,8 +200,59 @@ ${PYTHON} -m venv -- ${PWNDBG_VENV_PATH}
 source ${PWNDBG_VENV_PATH}/bin/activate
 poetry install
 
+# Function to detect the user's shell and return the configuration file path
+get_shell_config_file() {
+    echo "in func 1"
+    case "$SHELL" in
+        */bash)
+            echo "$HOME/.bashrc"
+            ;;
+        */zsh)
+            echo "$HOME/.zshrc"
+            ;;
+        */fish)
+            echo "$HOME/.config/fish/config.fish"
+            ;;
+        */csh | */tcsh)
+            echo "$HOME/.cshrc"
+            ;;
+        */ksh)
+            echo "$HOME/.kshrc"
+            ;;
+        *)
+            echo "Unsupported shell. You will need to manually add your shell config file."
+            return 1
+            ;;
+    esac
+}
+
+# Function to persist the environment variable in the appropriate shell configuration file
+persist_env_var() {
```

Comment:
We dont't want any of this. Let's just inform user on setup they can disable auto update checks with a given envvar and let the user to add it or not by themselves.

Reply:
I.e. we don't want to handle sh/bash/zsh/fish/bazilion other shells; the users can do it on their own

Reply:
Sure I got it. Will complete it by today.